packages/roslyn/package.yaml

```
@@ -0,0 +1,30 @@
+---
+name: roslyn
+description: |
+  The Roslyn .NET compiler provides C# and Visual Basic languages with rich code analysis APIs.
+homepage: https://github.com/dotnet/roslyn
+licenses:
+  - MIT
+languages:
+  - C#
+categories:
+  - LSP
+
+source:
+  id: pkg:github/Crashdummyy/roslynLanguageServer@4.12.0-1.24351.5
```

Any update on this ?

Btw the cleanup process removed this release by now.

```suggestion
  id: pkg:github/Crashdummyy/roslynLanguageServer@4.12.0-1.24373.10
```